components/benefit_card_header.js

```diff
@@ -27,84 +28,38 @@ const headerDesc = css`
     font-size: 12px;
   }
 `;
-const headerUrl = css`
-  color: #006cc9;
-`;
 
 export class BenefitCardHeader extends Component {
-  logExit = url => {
-    logEvent("Exit", url);
-  };
-
   componentDidMount() {
     this.forceUpdate();
   }
 
-  benefitTitle = benefit => {
-    return this.props.t("current-language-code") === "en"
-      ? benefit.vacNameEn
-      : benefit.vacNameFr;
-  };
-
-  benefitUrl = benefit => {
-    return this.props.t("current-language-code") === "en"
-      ? benefit.benefitPageEn
-      : benefit.benefitPageFr;
-  };
-
-  get_benefit_a_elements = parentBenefits => {
-    let a_elements = parentBenefits.map((b, i) => (
-      <a
-        key={"a" + i}
-        className={headerUrl}
-        href={this.benefitUrl(b)}
-        target="_blank"
-        rel="noopener noreferrer"
-        onClick={() => this.logExit(this.benefitUrl(b))}
-      >
-        {this.benefitTitle(b)}
-      </a>
-    ));
-
-    let a_elements_with_ors = [];
-    a_elements.forEach((value, index) => {
-      if (a_elements.length - 1 !== index) {
-        a_elements_with_ors = a_elements_with_ors.concat([
-          value,
-          <span key={"b" + index}> {" " + this.props.t("index.or")} </span>
-        ]);
-      } else {
-        a_elements_with_ors = a_elements_with_ors.concat([
-          value,
-          <span key={"c" + index}> </span>
-        ]);
-      }
-    });
-    return a_elements_with_ors;
-  };
-
   render() {
     const { t, benefit, benefits } = this.props;
     const parentBenefits = benefits.filter(
       b => b.childBenefits && b.childBenefits.includes(benefit.id)
     );
-
-    if (
+    const includeParentInfo =
       parentBenefits.length > 0 &&
-      benefit.availableIndependently === "Requires Gateway Benefit"
-    ) {
+      benefit.availableIndependently === "Requires Gateway Benefit";
+    const includeImportantInfo =
+      benefit.noteEn !== undefined && benefit.noteFr !== undefined;
```

Comment:
We could use `benefit.noteEn && benefit.noteFr` here to also catch the case where one of them is "".